### PR TITLE
Add New Event button

### PR DESCRIPTION
**Pull request by abdellani**
#### Issue addressed

fixes #

#### Screenshots (if appropriate):

Screenshot from 2019-03-26 08-52-12

### CODE REVIEW

**Comment by mattwr18** [comment]
would you mind taking a screenshot @abdellani... the code looks good and I'm sure it's all good, but if I could see it, then I can give a review and someone can merge it

**Comment by mattwr18** [comment]
Screenshot at 2019-03-25 23:50:34

File: src/containers/EventsList.js [hunk 1]
```diff
@@ -25,7 +25,29 @@ export class EventsList extends Component {
...
+                 AgileVentures Events
+                </Header>
+              </Grid.Column>
+              <Grid.Column width={1}>
+                <a href='events/new'>
+                  <Popup
+                    position='right center'
+                    trigger={
+                      <Button basic style={{ marginTop: '25px' }}>
+                        <Icon name='plus' id='new-proj-icon' />
```
Comment (mattwr18) [inline src/containers/EventsList.js]: Can you please change the id to be unique to the `new-event-icon`?

**Review by mattwr18 — CHANGES_REQUESTED** [review]

File: src/containers/EventsList.js (hunk 1 above)
Comment (abdellani) [reply src/containers/EventsList.js]: I updated the `id`

**Review by mattwr18 — APPROVED** [review]

**Pull request merged**